recipe/meta.yaml

```diff
@@ -58,28 +57,95 @@ requirements:
     - pyparsing
     #- py2cairo  # [linux and py2k]
     - libpng
-    - pyqt        # [not osx]
     - tk        # [linux]
     - functools32  # [py2k]
     - subprocess32  # [py2k and unix]
     - icu       # [linux]
     - tornado
     - kiwisolver
 
-test:
-  imports:
-    - matplotlib
-    - matplotlib.pyplot
-    - matplotlib._image
-    - matplotlib._path
-    - matplotlib._png
-    - matplotlib._tri
-    - matplotlib.backends._backend_agg
-    - matplotlib.ft2font
-    - matplotlib.ttconv
-    - matplotlib.backends._tkagg  # [not win]
-    - mpl_toolkits
-    - pylab
+outputs:
+  - name: matplotlib-base
+    script: install.sh  # [not win]
```

Comment:
Thanks for doing this.

please see https://github.com/conda-forge/matplotlib-feedstock/pull/196 for how to get rid of this whole duplicated section.

Reply:
@hmaarrfk, see 4d8eb86.

Reply:
Cool stuff Much easier to read